### PR TITLE
[raylib] update to 5.5

### PR DESCRIPTION
**Pull request by erikyuzwa**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [x] The "supports" clause reflects platforms that may be fixed by this new version.
- [ ] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [ ] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/raylib/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "raylib",
-  "version": "5.0",
+  "version": "5.5",
   "port-version": 2,
```
Comment (MonicaLiu0311) [inline ports/raylib/vcpkg.json:4]:
```suggestion
```

Comment (MonicaLiu0311) [reply ports/raylib/vcpkg.json:4]: 1. If a version is released upstream, update the `"version"` field in `vcpkg.json` and count `"port-version"` from 0 (port-version is a field that records changes specific to vcpkg, 0 generally means deleting "port-version" line).
2. If the upstream has not released a new version, then after modifying other files, please accumulate the `"port-version"` value to record this modification.

Please read the documentation: [maintainer guide#versioning](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md#versioning).

**Review by BillyONeal — APPROVED** [review]

**Review by MonicaLiu0311 — APPROVED** [review]

**Comment by erikyuzwa** [comment]
thanks so much @BillyONeal and @MonicaLiu0311 !! 🔥 ❤️

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**